packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.types.ts

```diff
@@ -82,8 +83,37 @@ export interface ITooltipHostProps extends React.HTMLAttributes<HTMLDivElement |
    */
   closeDelay?: number;
 
+  /**
+   * Call to provide customized styling that will layer on top of the variant rules.
+   */
+  styles?: IStyleFunctionOrObject<ITooltipHostStyleProps, ITooltipHostStyles>;
+
+  /**
+   * Theme provided by High-Order Component.
+   */
+  theme?: ITheme;
+
   /**
    * Notifies when tooltip becomes visible or hidden, whatever the trigger was.
    */
   onTooltipToggle?(isTooltipVisible: boolean): void;
 }
+
+export interface ITooltipHostStyleProps {
+  /**
+   * Accept theme prop.
+   */
+  theme: ITheme;
+
+  /**
+   * Accept optional classNames for the host wrapper
+   */
+  className?: string;
+}
+
+export interface ITooltipHostStyles {
+  /**
```

Comment:
as a convention, the top level is always root. I understand the idea being using 'host', but I fear it'd just cause confusion.